boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -172,10 +183,11 @@ arduino_serial: &uart1 {
 	};
 };
 
-&spi4 {
+arduino_spi: &spi4 {
 	compatible = "nordic,nrf-spim";
 	status = "okay";
-	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>,
+	           <&arduino_header 16 GPIO_ACTIVE_LOW>;
```

Comment:
```suggestion
	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>,
	           <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
```
and adjust addresses in `sdhc@0` and `cs47l63@1` below. `arduino_spi` users (e.g. shields) will likely use CS #0, not #2.

Reply:
Hi @anangl - could you please explain to a 'devicetree novice' like me? :)